### PR TITLE
Avatar & badge

### PR DESCRIPTION
**Pull request by msdeshmukh009**
orionui

### CODE REVIEW

**Review by horsemaker — APPROVED** [review]
The UI looks clean Mahesh! Also, the responsive hamburger menu felt glass-like. 👌

Though the avatars crunched in mobile view. Don't know if you are yet to tackle that problem.

**Comment by msdeshmukh009** [comment]
> The UI looks clean Mahesh! Also, the responsive hamburger menu felt glass-like. 👌
> 
> Though the avatars crunched in mobile view. Don't know if you are yet to tackle that problem.

Thanks! Can you please elaborate on the avatar problem?  Avatar is looking like this. What do you suggest I should improve?
avatar

File: docsite-css/component-display-conatiner.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.component-display-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  border: 2px solid rgb(221, 217, 217);
```
Comment (nshanbhag20001412) [inline docsite-css/component-display-conatiner.css:5]: you can add color variable here

**Review by nshanbhag20001412 — APPROVED** [review]
Rest looks good to me! Nice work!

File: docsite-css/component-display-conatiner.css (hunk 1 above)
Comment (msdeshmukh009) [reply docsite-css/component-display-conatiner.css:5]: Thanks! I will do that right away

**Review by MiheerTamkhane — APPROVED** [review]
Very clean UI. I think Get Started btn is not working?

**Comment by msdeshmukh009** [comment]
> Very clean UI. I think Get Started btn is not working?

Thanks! and you are right get started button is not working yet 😄 will work on that.

**Pull request merged**